lightning_pose/data/utils.py

```diff
@@ -113,9 +172,9 @@ def get_loader(self) -> Union[torch.utils.data.DataLoader, dict]:
         if self.cond == "test":
             return self.data_module.test_dataloader()
 
-    @typechecked
     def verify_labeled_loader(
-        self, loader: Union[torch.utils.data.DataLoader, dict]
+        self,
+        loader: Union[torch.utils.data.DataLoader, dict]
```

Comment:
if we decide to make a typed dict of the supervised+unsupervised dataloaders, we can use it here too. but no rush.